README.md

```diff
@@ -33,6 +33,14 @@ philosophy](https://abseil.io/about/philosophy).
 `prototype-kicad-library-ci` uses the [GitHub
 flow](https://guides.github.com/introduction/flow/) workflow.
 
+## Git Config Install Instructions
+To install this repository's Git configuration, execute the `install` script located in the
```

Comment:
Please change `Git configuration` to `Git config entries` which more accurately describes what the `git/config/install` script does (add entries to the repository's `.git/config` file), and matches the script's comment block.
```suggestion
To install this repository's Git config entries, execute the `install` script located in the
```

Please use a column limit of 90 (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide). Couldn't include this in the suggested change due to a possible bug in the raw markdown viewer (can't select multiple lines).